packages/backend/src/app.ts

```diff
@@ -216,6 +216,7 @@ export class App {
       }
     })
 
+    await this.apolloServer.start()
```

Comment:
This wasn't required in the old version?

Reply:
@wilsonianb  Yep, this was changed in the new version: https://www.apollographql.com/docs/apollo-server/api/apollo-server/#start

> Always call await server.start() before calling server.applyMiddleware and starting your HTTP server. This allows you to react to Apollo Server startup failures by crashing your process instead of starting to serve traffic.